modular_skyrat/modules/importedvendors/code/vendors.dm

```diff
@@ -0,0 +1,182 @@
+/obj/effect/spawner/random/vending/snackvend
+	loot = list(
+		/obj/machinery/vending/imported,
+		///obj/machinery/vending/imported/panslav,
+		/obj/machinery/vending/imported/yangyu,
+		/obj/machinery/vending/imported/mothic,
+		/obj/machinery/vending/imported/tizirian,
+	)
+
+/obj/machinery/vending/imported
+	name = "NT Sustenance Supplier"
+	desc = "A vending machine serving up only the finest of human college student food."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_vendors.dmi'
+	icon_state = "ntfood"
+	panel_type = "panel15"
+	light_mask = "ntfood-light-mask"
+	light_color = LIGHT_COLOR_LIGHT_CYAN
+	product_slogans = "Caution, contents may be selling hot!;Look at these low prices!;Hungry? Me too, wait no you didn't hear that!"
+	product_categories = list(
+		list(
+			"name" = "Snacks",
+			"icon" = "cookie",
+			"products" = list(
+				/obj/item/food/peanuts/random = 6,
```

Comment:
Out of curiosity, does this choose one type-per-vendor on initialize, or is each vended item a random type?
No review importance just curious

Reply:
The vended item is a random one of any of the possible flavors for each item vended